api/tests/unit/infrastructure/datasources/airtable/area-datasource_test.js

```diff
@@ -18,4 +21,30 @@ describe('Unit | Infrastructure | Datasource | Airtable | AreaDatasource', () =>
     });
   });
 
+  describe('#findByRecordIds', () => {
+
+    it('should return an array of matching airtable area data objects', async function() {
+      // given
+      const rawArea1 = areaRawAirTableFixture('RECORD_ID_RAW_AREA_1');
+      const rawArea2 = areaRawAirTableFixture('RECORD_ID_RAW_AREA_2');
+      const rawArea3 = areaRawAirTableFixture('RECORD_ID_RAW_AREA_3');
+      const rawArea4 = areaRawAirTableFixture('RECORD_ID_RAW_AREA_4');
+
+      const records = [rawArea1, rawArea2, rawArea3, rawArea4];
+      sinon.stub(airtable, 'findRecords').callsFake(makeAirtableFake(records));
+
+      // when
+      const foundAreas = await areaDatasource.findByRecordIds([
+        rawArea1.fields['id persistant'],
+        rawArea2.fields['id persistant'],
+        rawArea4.fields['id persistant'],
+      ]);
+
+      // then
+      expect(foundAreas).to.be.an('array');
+      expect(_.map(foundAreas, 'id')).to.deep.equal([rawArea1.fields['id persistant'], rawArea2.fields['id persistant'], rawArea4.fields['id persistant']]);
+      expect(airtable.findRecords).to.have.been.calledWith('Domaines');
```

Comment:
```suggestion
      const expectedAreas = [
        rawArea1.fields['id persistant'],
        rawArea2.fields['id persistant'],
        rawArea4.fields['id persistant'],
      ]
      // when
      const foundAreas = await areaDatasource.findByRecordIds(expectedAreas);

      // then
      expect(foundAreas.map(({ id }) => id)).to.deep.equal(expectedAreas);
```

A reproduire dans les autres data sources